test/helpers/router.js

```diff
@@ -34,11 +34,22 @@ var $Router = {
       this.sails.router.bind.apply(this.sails.router, args);
     });
 
+    it('unbinds the given route', function() {
+      this.sails.router.unbind.should.be.ok;
```

Comment:
Hmm, yes it is. The `.should.be.false` and `.should.be.true`s below also need to change then.